sdk/management/README.md

```diff
@@ -79,14 +79,16 @@ With above configuration, `azure` client can be authenticated by following code:
 
 ```java
 AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
-TokenCredential credential = new DefaultAzureCredentialBuilder()
+TokenCredential credential = new EnvironmentCredentialBuilder()
```

Comment:
is DefaultAzureCredentialsBuilder no longer supported?

Reply:
`DefaultAzureCredentialsBuilder` is still there, but 1.0.9 does not have `authorityHost` method, which means it does not work for Cloud other than global Azure (e.g. Azure US gov, Azure China, Azure Germany).

I am not sure whether we should still use `DefaultAzureCredentialsBuilder` in readme (which then not works for other clouds), or we should use the one that supports these clouds.

Reply:
hi @weidongxu-microsoft do you know why DefaultAzureCredentailsBuilder not supporting authorityHost? it's `Default`.

Reply:
I will check with Jianghao.

Reply:
1.1.0 will be released in 2 weeks, which should have `authorityHost` by then.

I will now only change the sample code to remove `authorityHost` at time being, then add it back after azure-identity 1.1.0 release.

Reply:
thanks @weidongxu-microsoft